### DISCUSSION TITLE
I'm a newbie who doesn't really understand BitTorrent and I'm not sure how to use this app

### DISCUSSION THREAD

**Discussion by SheepCommander** [Bartuzen/qBitController#50] [Q&A] [answered]
I want to torrent some media that's no longer being streamed, and I wanna do it on my phone because my PC is out of space

is this the right app for that?

**Comment by Bartuzen** [comment, marked as answer]
This app connects to qBittorrent installed on a PC, it doesn't download torrent to your phone. For that you can use [LibreTorrent](https://github.com/proninyaroslav/libretorrent/).

**Reply by SheepCommander** [reply]
thanks for the link!
